examples/trivial-digital-ocean.nix

```diff
@@ -0,0 +1,12 @@
+{
+  resources.sshKeyPairs.ssh-key = {};
+
+  machine = { config, pkgs, ... }: {
+    services.nginx.enable = true;
+    services.openssh.enable = true;
+
+    deployment.targetEnv = "digital-ocean";
+    deployment.digital-ocean.region = "ams2";
```

Comment:
Maybe this should be `digitalOcean`, since we generally use camelCase in option names.